tools/xs-opam-ci.env

```diff
@@ -1,4 +1,5 @@
 export OCAML_VERSION="4.08"
 export DISTRO="debian-unstable"
 export BASE_REMOTE="https://github.com/xapi-project/xs-opam.git"
+export BASE_REMOTE_BRANCH="release/stockholm/lcm"
```

Comment:
Why do we need this? This refers to the Jenkins build system and I don't understand the relevance here.

Reply:
This is needed to test other repositories against this branch when using travis.

See https://github.com/xapi-project/xen-api/pull/4132

Reply:
I didn't pay attention to the target branch